lib/dashing/app.rb

```diff
@@ -120,16 +119,6 @@ def protected!
   end
 end
 
-Thin::Server.class_eval do
-  def stop_with_connection_closing
-    Sinatra::Application.settings.connections.dup.each(&:close)
-    stop_without_connection_closing
-  end
-
-  alias_method :stop_without_connection_closing, :stop
-  alias_method :stop, :stop_with_connection_closing
-end
-
```

Comment:
Here's the PR that introduced that change for some context: https://github.com/Shopify/dashing/pull/383

I am aware that this is thing dependant, and I do want to take it out.